docs/modules/ROOT/pages/platform-deploy.adoc

```diff
@@ -0,0 +1,87 @@
+= Using with OpenZeppelin Platform
+
+The Hardhat Upgrades package can use the OpenZeppelin Platform for deployments instead of ethers.js.
+
+[[configuration]]
+== Configuration
+
+Create a deployment environment on the OpenZeppelin Platform and provide the Team API Key and secret in your `hardhat.config.js` or `hardhat.config.ts` file under `platform`:
+
+[source,js]
+----
+module.exports = {
+  platform: {
+    apiKey: process.env.API_KEY,
+    apiSecret: process.env.API_SECRET,
+  }
+}
+----
+
+[[usage]]
+== Usage
+
+When using the xref:api-hardhat-upgrades.adoc[Hardhat Upgrades API functions], enable OpenZeppelin Platform deployments using any of the ways below.
```

Comment:
Let me know what you think of this:
- If you want to use the same scripts on local devnets and Platform, use `upgrades.<fn>` with `deploy: true` in the config that you use with Platform.
- If you only run your scripts on Platform, use `platform.<fn>` for clarity.

Reply:
I would say to use `platform.<fn>` when the user wants to make sure Platform is used and wants to see an error otherwise, but mostly they should use the first option you describe.